games/Doomed.js

```diff
@@ -0,0 +1,1417 @@
+/*
+@title: Doomed
+@author: m5kro
+@tags: [singleplayer, shooter]
```

Comment:
the tags need quotation marks around them

Reply:
Added!